Deeploy/Targets/Snitch/TypeCheckers.py

```
@@ -0,0 +1,25 @@
+from typing import List, Sequence, Type
+
+from Deeploy.CommonExtensions.TypeCheckers.SignPropTypeChecker import SignPropTypeChecker
+from Deeploy.DeeployTypes import OperatorRepresentation, Pointer, VariableBuffer
+
+
+class iNoNormChecker(SignPropTypeChecker):
+
+    def __init__(self, input_types: Sequence[Type[Pointer]], output_types: Sequence[Type[Pointer]]):
+        super().__init__(input_types, output_types)
+
+    def _inferNumLevels(self, inputs: List[VariableBuffer],
```

Ruff (0.14.2)

Unused method argument: `inputs`

(ARG002)